compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/packaging/Resolution.java

```diff
@@ -9,11 +10,11 @@
  * Had to wrap in a class since you can't return multiple items form a java method.
  */
 public class Resolution {
-    public static final Resolution NOT_FOUND = new Resolution(null, null);
+    public static final Resolution NOT_FOUND = new Resolution(null, null, null);
```

Comment:
We don't need to pass last null as it is not used

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-lang/pull/10442/commits/fa74255bc087ed97ca3d7fb2f28539751a69e5f0